novelai-api/api.go

```diff
@@ -161,6 +177,7 @@ func NewGenerateParams() NaiGenerateParams {
 	repPenSlope := 6.57
 	banBrackets := true
 	badWordsIds := make([][]uint16, 0)
+	logitBiasIds := make([][]float32, 1)
```

Comment:
Is there a reason why we're creating an array size of `1` rather than starting with an empty array of `0`?